module-services/service-desktop/Sync.cpp

```diff
@@ -57,6 +57,16 @@ Sync::CompletionCode Sync::PrepareSyncPackage(sys::Service *ownerService, std::f
 bool Sync::RemoveSyncDir(const std::filesystem::path &path)
 {
     /* prepare directories */
+    const auto tempPath = purefs::dir::getTemporaryPath();
+    if (std::filesystem::exists(tempPath) and !std::filesystem::is_directory(tempPath)) {
+        LOG_WARN("%s should be a directory, removing...", tempPath.c_str());
+        if (!std::filesystem::remove(tempPath)) {
+            LOG_ERROR("Failed to remove %s!", tempPath.c_str());
+            return false;
+        }
+        return true;
+    }
+
```

Comment:
do you know why the `temp` catalog becomes a file? I think we should find the source of that issue instead of fixing it this way...

Reply:
We don't know. 
The filesystem was OK. fsck didn't report any errors. The logs also don't contain anything connected to this situation.
So I guess we don't fix it because we don't have any reproduction steps just a phone with such a condition :D